app/presenters/search_presenter.rb

```diff
@@ -11,7 +11,11 @@ def to_json
       description: document.summary,
       link: document.base_path,
       indexable_content: indexable_content,
-      organisations: organisation_slugs,
+      # FIXME: This line was dependent on a method called #organisation_slugs,
+      # which in turn used publishing_api.get_content to map org content_ids to
+      # organisation slugs. It was stubbed in many places in spec/, but that method
+      # call always returned a blank list in the real world.
+      organisations: [],
```

Comment:
There won't be any user impact, based on the assumption that the Finding Things work will go to production before this tool. If this assumption is no longer true, we will revert this change. I'd prefer to not leave any placeholders in, as that would necessitate another change to clean them out later.